### ISSUE TITLE
Create dataset loader for UIT-ViOCD

### ISSUE THREAD

**Issue by SamuelCahyawijaya** [SEACrowd/seacrowd-datahub#305]
Dataloader name: `uit_viocd/uit_viocd.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?uit_viocd

|     Dataset| uit_viocd |
|-------------|---|
| Description | The UIT-ViOCD dataset includes 5,485 reviews e-commerce sites across four categories: fashion, cosmetics, applications, and phones. Each review is annotated by humans, assigning a label of 1 for complaints and 0 for non-complaints. The dataset is divided into training, validation, and test sets, distributed approximately in an 80:10:10 ratio. |
| Subsets | - |
| Languages | vie |
| Tasks | Text Classification |
| License | Unknown (unknown) |
| Homepage | https://huggingface.co/datasets/tarudesu/ViOCD |
| HF URL | https://huggingface.co/datasets/tarudesu/ViOCD |
| Paper URL | https://ebooks.iospress.nl/DOI/10.3233/FAIA210058 |

**Comment by Alex-HaochenLi** [comment]
#self-assign

**Comment by Alex-HaochenLi** [comment]
Hi @holylovenia, @SamuelCahyawijaya, @sabilmakbar, please add a 'pr-ready' tag to this issue.
